src/pages/SignUp/SignUp.js

```diff
@@ -1,21 +1,71 @@
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
+import { useReducer } from "react";
 import "./SignUp.css";
+import axios from "axios";
+import { useAuth } from "../../contexts/auth-context";
+import { signUpReducer } from "../../reducer";
+
 export function SignUp() {
+  const initialSignUpValue = {
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+  };
+
+  const [signUpState, signUpDispatch] = useReducer(
+    signUpReducer,
+    initialSignUpValue
+  );
+  // console.log(signUpState)
```

Comment:
remove the commented code before pushing

Reply:
will do that

Reply:
removed